rules/cloud/aws/aws_iam_s3browser_templated_s3_bucket_policy_creation.yml

```diff
@@ -0,0 +1,33 @@
+title: AWS IAM S3Browser Templated S3 Bucket Policy Creation
+id: db014773-7375-4f4e-b83b-133337c0ffee
+status: experimental
+description: Detects S3 Browser utility creating Inline IAM Policy containing default S3 bucket name placeholder value of <YOUR-BUCKET-NAME>.
+references:
+    - https://permiso.io/blog/s/unmasking-guivil-new-cloud-threat-actor
+author: daniel.bohannon@permiso.io (@danielhbohannon)
+date: 2023/05/17
+modified: 2023/05/17
+tags:
+    - attack.execution
+    - attack.t1059.009
+    - attack.persistence
+    - attack.t1078.004
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection_source:
+        eventSource: iam.amazonaws.com
+        eventName: PutUserPolicy
+    filter_tooling:
+        userAgent|contains: 'S3 Browser'
+    filter_policy_resource:
+        requestParameters|contains: '"arn:aws:s3:::<YOUR-BUCKET-NAME>/*"'
+    filter_policy_action:
+        requestParameters|contains: '"s3:GetObject"'
+    filter_policy_effect:
+        requestParameters|contains: '"Allow"'
+    condition: selection_source and filter_tooling and filter_policy_resource and filter_policy_action and filter_policy_effect
```

Comment:
You're using an `and` so the `filter_*` are not meant to exclude something but rather to include it? If so then you can group them in the same selection.

Reply:
I assumed its all "and" based on the logic of the blog